FAQ.md

```diff
@@ -0,0 +1,63 @@
+---
+status: draft
+---
+
+# Frequently Asked Questions (FAQs)
+
+## Why create a new foundation?
+
+There are already several foundations which host open source projects,
+and each has its own focus and requirements.
+The CF is a home for self-governing, code-centric open source projects.
+Each open source project has its own unique needs and goals,
+and not every foundation is the right fit for every project.
+
+There are many critical application frameworks and libraries within the open source ecosystem that are not currently part of a foundation.
+The CF is a new option for projects to consider when looking for a foundation to call home.
+Helping to strengthen the open source ecosystem as a whole by ensuring more projects are sustainable in the long term.
```

Comment:
@gavinking said:

>  Why not just remove all that and have a positive list of "differentiators", which doesn't need to be explicit about who each point differentiates us from.

For the record, I had made the same suggestion over emails yesterday. I feel that calling out specific points of each foundation is unnecessary and could be abused to ignite unnecessary debate - we don't need this to be super specific like a laundry list of complaints.

Back to the specific wording of the FAQ: it feels like line 17 was meant to be a continuation of the previous line? Perhaps only separated by a comma?

Reply:
I've fixed the sentence continuation @Sanne, thanks